ogr/services/github/project.py

```diff
@@ -594,6 +594,7 @@ def _pr_from_github_object(github_pr: GithubPullRequest) -> PullRequest:
             source_branch=github_pr.head.ref,
             target_branch=github_pr.base.ref,
             created=github_pr.created_at,
+            diff_url="/".join([github_pr.html_url, "files"]),
```

Comment:
Yes, I would go with this one.

What about using `patch_url` for the other one (e.g.https://patch-diff.githubusercontent.com/raw/packit-service/ogr/pull/1.diff).

Reply:
What you mean but that, add one more field patch_url? Or use .diff then /files?

Reply:
- I agree with the `/files`.
- Just an idea to provide also `.diff` in some way. (e.g. `project.patch_url`)
  - We can create a new issue for that.